tests/internal/scripts/ccuda/cupy_ravel.py

```diff
@@ -0,0 +1,25 @@
+from pyccel.decorators import kernel, types
+from pyccel import cuda
+import cupy as cp
+
+@kernel
+@types('int[:]')
+def func(a):
```

Comment:
can we have an example that is changing some elements of the array passed

Reply:
Sure i'll add that.

Reply:
done in [dcdc81f](https://github.com/pyccel/pyccel/pull/1222/commits/dcdc81fb9230c8977b7563a658c1bedc2d1037b1)